### ISSUE TITLE
Need gui to change file type associations

### ISSUE THREAD

**Issue by vasyugan** [nextcloud/server#29017]
### How to use GitHub

**Is your feature request related to a problem? Please describe.**
I would like to have Collabora and OnlyOffice installed in parallel and have them handle OpenDocument and OOXML files respectively, however [as soon as I install collabora, it also registers itself as the default for all office file types](https://github.com/nextcloud/richdocuments/issues/1400), putting OnlyOffice out of work. 

**Describe the solution you'd like**
As in all desktop environments, there should be an area in Nextcloud where the admin can set file type handlers and determine their priorities. Ideally, there also would be a user level setting, too. 
 
**Describe alternatives you've considered**
Collabora could instead have its menu for file types, but this would be introducing clutter. Better to have one central place.

**Comment by szaimen** [comment]
Duplicate of https://github.com/nextcloud/richdocuments/issues/1400

**Comment by vasyugan** [comment]
> Duplicate of [nextcloud/richdocuments#1400](https://github.com/nextcloud/richdocuments/issues/1400)

This is not a duplicate. Instead of having each app installing its own file types menu, there should be _one central place_ See my suggestion above.

**Comment by szaimen** [comment]
Sry I wasn't clear enough: let's track this in https://github.com/nextcloud/richdocuments/issues/1400

**Comment by vasyugan** [comment]
> Sry I wasn't clear enough: let's track this in [nextcloud/richdocuments#1400](https://github.com/nextcloud/richdocuments/issues/1400)

I my view, this makes little sense, because we don't want every app to ship their own file types menu. We don't want changes to the code of Collabora but of the Nextcloud core. Else we need a new app.  But what we don't need is every app coming with its own configuration interface. Do you disagree? I'll leave this closed for now, waiting for your response, but I would be highly surprised if you have good reasons to keep this a collabora issue.

**Comment by Spartachetto** [comment]
Isn't this a duplicate of https://github.com/nextcloud/server/issues/8105 ? Basically this means that I agree with @vasyugan in seeing this as a server issue
